src/renderer/domains/collectives/referendum/resource.ts

```diff
@@ -54,9 +64,11 @@ async function parseProposal(proposal: FrameSupportPreimagesBounded, api: ApiPro
 
     if (struct.section === 'polkadotXcm' && struct.method === 'send') {
       //todo learn how to read arguments (multilocation chainId and call)
-
       return {
         type: 'Whitelist',
+        proposalHex,
+        proposalHash: blake2AsHex(proposalHex),
+        proposalJSON: struct.toJSON(),
```

Comment:
do we need this? `toJSON` output is weirdly structured, i don't think it's usable in any way